superduperdb/backends/base/query.py

```diff
@@ -145,6 +145,33 @@ def _set_db(r, db):
             parts.append((part, part_args, part_kwargs))
         self.parts = parts
 
+    def listeners(self, ):
```

Comment:
This is about detecting which `Listener` applies to the query right?

Reply:
I will rename it to dependencies